uportal-war/src/main/java/org/jasig/portal/api/permissions/ApiPermissionsService.java

```diff
@@ -79,10 +79,10 @@
             	}
             }
         }
-        
+
         if (includeInherited) {
             IGroupMember member = GroupService.getGroupMember(authP.getKey(), authP.getType());
-            for (@SuppressWarnings("unchecked") Iterator<IEntityGroup> iter = member.getAncestorGroups(); iter.hasNext();) {
+            for (Iterator<IEntityGroup> iter = member.getAncestorGroups().iterator(); iter.hasNext();) {
                 IEntityGroup parent = iter.next();
 
```

Comment:
Help yourself!  Send a PR to drewwills/UP-4668... it will automatically become a party of this PR when merged.  I will get to it if you don't, though.